mindmeld/components/nlp.py

```diff
@@ -1083,6 +1083,28 @@ def nbest_transcripts_enabled(self):
     def nbest_transcripts_enabled(self, value):
         self._nbest_transcripts_enabled = value
 
+    def extract_entities(self, label_set=None):
+
+        # Create the entity processors
+        _processors = Bunch()
+        entity_types = self.entity_recognizer.get_entity_types(label_set=label_set)
+        for entity_type in entity_types:
+
+            if entity_type in self._children:
```

Comment:
Its unclear why we do this

Reply:
Corrected to `if entity_type in _processors` from `if entity_type in self._children`